dds_cli/project_creator.py

```diff
@@ -87,6 +89,24 @@ def create_project(
                 LOG.error(error)
                 return created, created_project_id, user_addition_statuses, error
 
+            warning_message = response.json().get("warning")
+            if warning_message:
+                proceed_creation = rich.prompt.Confirm.ask(
```

Comment:
This should have a no_prompt option as well.

Reply:
Done!